system/jlib/jstatcodes.h

```diff
@@ -175,9 +178,8 @@ enum StatisticKind
     StSizeSpillFile,
     StCycleSpillElapsedCycles,
     StCycleSortElapsedCycles,
-
-    // Stranding stats - on edge
-    StNumStrands,
+    StNumStrands,                       // Stranding stats - on edge
+    StNumCalls,
```

Comment:
NumEvaluations might be a more general/reusable tag - e.g., if this is used for general expressions as well.